src/utils/time.ts

```diff
@@ -31,7 +31,7 @@ export function getCurrentTimeInHighPrecision(): string {
  * @returns string
  */
 export function createTimestamp(
-  year: number, month: number, day: number, hour: number, minute: number, second: number, millisecond: number, microsecond: number
+  year?: number, month?: number, day?: number, hour?: number, minute?: number, second?: number, millisecond?: number, microsecond?: number
```

Comment:
Wrap in options so that it is always clear what properties are being set, so there is less chance for misinterpretation or usage.
```suggestion
  options: { year?: number, month?: number, day?: number, hour?: number, minute?: number, second?: number, millisecond?: number, microsecond?: number }
```